homeassistant/components/tibber/sensor.py

```diff
@@ -21,12 +29,16 @@
 PARALLEL_UPDATES = 0
 
 
-async def async_setup_entry(hass, entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: Callable[[Sequence[Entity], bool], None],
```

Comment:
I feel this type will be needed quite often. Maybe add it to `homeassistant.helpers.typing`?

The exact type can be described using `typing_extensions.Protocol`:
```python
class AddEntitiesCallback(Protocol):
    """Protocol type for async_setup_entry callback"""

    def __call__(
        self, new_entities: Iterable[Entity], update_before_add: bool = False
    ) -> None:
        ...

Reply:
I would consider that out of scope for this PR.